package.json

```diff
@@ -17,6 +17,8 @@
     "format": "lerna run format --stream",
     "format:docs": "prettier --write docs/**/*.html",
     "test:all": "lerna run test:all --stream",
+    "validate-spec": "npx swagger-cli validate docs/openapi/openapi.yml",
+    "preserve": "npx swagger-cli bundle docs/openapi/openapi.yml -o docs/openapi/openapi.json",
```

Comment:
please document this command or remove it.

Reply:
This is great actually - was just working through something similar on my side - please do add some docs on this

Reply:
@OR13 @mprorock Readme.md has been updated with some documentation regarding these commands.